js/ui/tree_view/ui.tree_view.base.js

```diff
@@ -1597,15 +1598,15 @@ var TreeViewBase = HierarchicalCollectionWidget.inherit({
     },
 
     _expandFocusedContainer: function() {
-        var $focusedNode = this.option("focusedElement");
-        if(!$focusedNode || $focusedNode.hasClass(IS_LEAF)) {
+        var $focusedNode = $(this.option("focusedElement"));
+        if(!$focusedNode.length || $focusedNode.hasClass(IS_LEAF)) {
             return;
         }
 
         var $node = $focusedNode.find("." + NODE_CONTAINER_CLASS).eq(0);
 
         if($node.hasClass(OPENED_NODE_CONTAINER_CLASS)) {
-            this.option("focusedElement", this._nextItem(this._findNonDisabledNodes(this._nodeElements())));
+            this.option("focusedElement", getPublicElement(this._nextItem(this._findNonDisabledNodes(this._nodeElements()))));
```

Comment:
It is necessary to refactor

Reply:
Yes, it is. But are you sure that it should be done in this PR?

Reply:
Yes. You made this line more complex by adding another one method call

Reply:
It's a result of the auto replace, so I'm sure, that typically, we do not make refactoring in the same situations.
Of course, I improve this code since you find it already :)